### PR TITLE
iOS CI Update

### PR DESCRIPTION
**Pull request by paultiarks**
This runs tests on both iOS 12 & iOS 13 and adds a missing test case from the EventPublisher

### CODE REVIEW

File: platforms/apple/Sources/NimbusTests/CallJavascriptTests.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ class CallJavascriptTests: XCTestCase, WKNavigationDelegate {
     func loadWebViewAndWait(html: String = "<html><body></body></html>") {
         loadingExpectation = expectation(description: "web view loaded")
         webView.loadHTMLString(html, baseURL: nil)
-        wait(for: [loadingExpectation!], timeout: 10)
+        wait(for: [loadingExpectation!], timeout: 60)
```
Comment (threeve) [inline platforms/apple/Sources/NimbusTests/CallJavascriptTests.swift:41]: 😩

File: platforms/apple/Sources/NimbusTests/EventPublisherTests.swift [hunk 2]
```diff
@@ -64,6 +64,23 @@ class EventPublisherTests: XCTestCase {
         waitForExpectations(timeout: 1, handler: nil)
     }
 
+    func testDispatchTwice() {
```
Comment (threeve) [inline platforms/apple/Sources/NimbusTests/EventPublisherTests.swift:67]: Could you split this out into its own PR, it feels very out of place with what the focus of this PR is.

File: platforms/apple/Sources/NimbusTests/CallJavascriptTests.swift (hunk 1 above)
Comment (paultiarks) [reply platforms/apple/Sources/NimbusTests/CallJavascriptTests.swift:41]: It's truly a tragedy, but I don't know of an easier/cleaner way of solving the problem without introducing a bunch more risk.

**Comment by threeve** [comment]
Do we need to test on iOS 11 as well, since that's our current deployment target?

**Comment by threeve** [comment]
Or should we consider bumping deployment target to 12?

**Comment by paultiarks** [comment]
I'm not sure dropping iOS 11 is necessary at this point. We should probably be testing all three. The build and test task is already fairly large and takes a while. I think I'm going to create two new tasks, 1 that builds and tests on 11, and 1 that builds and tests on 12. Then I'm going to amend some of the changes already done here and leave the main job doing all of the coverage gathering/uploading.
